src/styles/theme/colors.ts

```diff
@@ -0,0 +1,144 @@
+export const baseColors = {
+    black: "#000000",
+    white: {
+        pure: "#ffffff",
+        transparent: "rgba(255, 255, 255, 0.65)",
+    },
+    // named darkest to lightest
+    dark: {
+        one: "#141219",
+        two: "#19171f",
+        three: "#1e1b25",
+        four: "#25222e", // fka three
+        five: "#2f2a3c", // fka four
+        six: "#3b3649", // fka charcoal gray, its purplish...
+        seven: "#4a4658",
+    },
+    // all "offwhites" now in the gray hierarchy
+    // lightest to darkest
+    gray: {
+        one: "#f7f7f7",
+        two: "#f4f4f4",
+        three: "#f2f2f2",
+        four: "#e7e7e7", // fka white six
+        five: "#d3d3d3",
+        six: "#d8d8d8", // fka three
+        seven: "#979797",
+        eight: "#737373",
+        nine: "#4a4a4a",
+    },
+    // lightest to darkest
+    purple: {
+        one: "#e7e4f2", // fka light
+        two: "#ddd9ec",
+        three: "#bab5c9",
+        four: "#d0c7ed",
+        five: "#d7c8ff",
+        six: "#b59ff6", // fka baby
+        seven: "#322759",
+        eight: "#2d224d",
+        nine: "#827aa3", // from allen master style guide
+        ten: "#8048f3", // from allen master style guide
+    },
+    blue: "#0094FF",
+    // brightest to dimmest
+    red: {
+        one: "#c23030",
+        two: "#d14040",
+    },
+    transparent: "transparent",
+};
+
+export const semanticColors = {
+    // bg colors
+    lightBg: baseColors.gray.one,
+    mediumBg: baseColors.dark.six,
+    grayBg: baseColors.gray.four,
+    darkBg: baseColors.dark.four,
+    lightPurpleBg: baseColors.purple.one,
+    // text colors
+    lightBgText: baseColors.dark.one,
+    lightBgActiveText: baseColors.dark.three,
+    darkBgText: baseColors.gray.nine,
+    whiteText: baseColors.white.pure,
+    // accent colors
+    primaryPurple: baseColors.purple.six,
+    activePurple: baseColors.purple.five,
+    primaryBlue: baseColors.blue,
+    primaryBorder: baseColors.gray.seven,
+    sliderColor: baseColors.gray.six,
+    buttonTextGray: baseColors.gray.five,
+    tagBg: baseColors.purple.nine,
+    tagText: baseColors.gray.nine,
+};
```

Comment:
I think these can/will continue to evolve a bit as we dial in all styling, but file now has a basic way of exporting base colors, these semantic colors, and then component specific assignments. Spreading all three into the `themeColors` export makes any layer the color hierarchy (base, semantic, component) available via styled components theme props.

Reply:
Would it make sense to not export base colors directly? That way you push development towards assigning specific semantic colors or component colors instead of relying on the hardcoded color directly.

Reply:
So, if I'm understanding you I think I do at the bottom of the file?
```
export const themeColors = {
    ...baseColors,
    ...semanticColors,
    ...componentColors,
};
```
I'm trying to balance both exporting all the most specific instances of the colors for use wherever they end up being needed, and the feedback I've received on previous PRs to consolidate anything that gets used repeatedly in similar contexts into these semantic variables.